dockerfiles/base/scripts/base/library.sh

```diff
@@ -74,56 +74,6 @@ server_is_booted() {
   fi
 }
 
```

Comment:
@TylerJewell should I remove that line? I don;t know how to detect where that method was used